lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,194 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
```

Comment:
Can you add `$ErrorActionPreference = 'Stop'` to this, would have helped you to pick up some of the errors below and keeps it more inline with other PS modules we have.

Reply:
Done.